source/routing/tests/get_stop_times_test.cpp

```diff
@@ -263,7 +263,7 @@ BOOST_AUTO_TEST_CASE(test_looping_frequency_for_calendar) {
     size_t headway_sec = 1000;
     b.frequency_vj("A", vj1_departure, 2001, headway_sec, "default_network", "1010", "", true, "vj1")
         (spa1, vj1_departure, vj1_departure)
-        ("useless_stop", 1000, 1000);
+        ("useless_stop", 87400, 87400);
```

Comment:
It sounds OK that we break on that... What's weird is it wasn't the case before.